parsers/CO.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import json
+import arrow
+
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import Optional
+from requests import Session
+
+#   PARSER FOR COLOMBIA / DEMAND-ONLY as of 2023-02-11 / 5-minute-granularity / returns demand data of the recent day
+#   MAIN_WEBSITE = https://www.xm.com.co/consumo/demanda-en-tiempo-real
+colombia_demand_URL = "https://serviciosfacturacion.xm.com.co/XM.Portal.Indicadores/api/Operacion/DemandaTiempoReal"
+
+TZ = "America/Bogota"  # UTC-5
+demand_list = []
+
+def fetch_consumption(
+    zone_key: str = "CO",
+    session: Session = Session(),
+    target_datetime: Optional[datetime] = None,
+    logger: Logger = getLogger(__name__),
+):
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates.")
+    with session.get(colombia_demand_URL, verify=False) as response:
+        demand_json = json.loads(response.content)
+        demand_pretty = demand_json['Variables'][0]['Datos']
```

Comment:
I would rename `demand_pretty` to something like `demand_data` just for future readability.

Reply:
fixed :)